### DISCUSSION TITLE
USB WifinLV-UW03

### DISCUSSION THREAD

**Discussion by zafracom** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#142] [Q&A]
I have mac os bigsure, i already installed driveer but, in apps cannot detect USB wireless 11N LV-UW03, does anyone have the same regarding this issue ?

**Comment by chris1111** [comment]
> I have mac os bigsure, i already installed driveer but, in apps cannot detect USB wireless 11N LV-UW03, does anyone have the same regarding this issue ?


This is not a Realtek, its  a Mediatek Drivers not support here
This is not a discussion, its an issue this will be deleted
See my github to find drivers for this adapter in Big Sur
